app/models/user.rb

```diff
@@ -38,6 +39,7 @@ class User < Common::RedisStore
 
   validates :uuid, presence: true
   validates :email, presence: true
+  validates :loa_current, presence: true
 
   # conditionally validate if user is LOA3
   with_options if: :loa3? do |user|
```

Comment:
@kreek can confirm what @robbiethegeek said.  Furthermore, ID.me is supposed to have this work done by Oct 14.  We will need to coordinate with them to see if this is a guaranteed attribute or not from there on out.  For now, I added a TODO comment.